### PR TITLE
(CL) Stake existing full range pos

### PR DESCRIPTION
**Pull request by jonator**
## What is the purpose of the change

If a user has a full range position in a superfluid enabled pool, this button allows them to stake it. As with balancer pools, it advertises the APR on the button, and includes superfluid styling. If a SF position already exists, it reuses the validator address.

### ClickUp Task

[ClickUp Task URL](PASTE_CLICKUP_TASK_URL_HERE)

## Brief Changelog

- "Go superfluid" button, with +APR
If not SF staked in this pool before and a full range position, the expanded view includes:
- "Go superfluid" button, with APR
- Opens select SF validator modal (reused from balancer)
- Triggers staking message

Try again with a fresh full range position with the above position staked:
- Validator selection should be skipped and tx should pop up

TODO
- [x] Validator selection modal CSS seemed to have been broken in the past. Fix it

### CODE REVIEW

File: packages/proto-codecs/src/codegen/osmosis/cosmwasmpool/v1beta1/genesis.ts [hunk 1]
```diff
@@ -97,3 +153,81 @@ export const GenesisState = {
     };
   },
 };
+export const PoolI_InterfaceDecoder = (
```
Comment (MaxMillington) [inline packages/proto-codecs/src/codegen/osmosis/cosmwasmpool/v1beta1/genesis.ts:156]: This section looks good, but one thing I notice is that there isn't really any error handling. Given what we have seen in the sentry, is it worth wrapping in an error handler?

File: packages/proto-codecs/src/codegen/osmosis/superfluid/tx.ts [hunk 2]
```diff
@@ -2452,3 +2495,225 @@ export const MsgAddToConcentratedLiquiditySuperfluidPositionResponse = {
...
+  decode(
+    input: _m0.Reader | Uint8Array,
+    length?: number
+  ): MsgLockExistingFullRangePositionAndSFStake {
+    const reader = input instanceof _m0.Reader ? input : new _m0.Reader(input);
+    const end = length === undefined ? reader.len : reader.pos + length;
+    const message = createBaseMsgLockExistingFullRangePositionAndSFStake();
+    while (reader.pos < end) {
+      const tag = reader.uint32();
+      switch (tag >>> 3) {
```
Comment (MaxMillington) [inline packages/proto-codecs/src/codegen/osmosis/superfluid/tx.ts:2531]: could the `tag` be negative? If so, you might want to use `>>`

**Review by MaxMillington — APPROVED** [review]
Lot's of scary messaging code, otherwise looks fine

File: packages/proto-codecs/src/codegen/osmosis/superfluid/tx.ts (hunk 2 above)
Comment (jonator) [reply packages/proto-codecs/src/codegen/osmosis/superfluid/tx.ts:2531]: This is generated 😅

File: packages/proto-codecs/src/codegen/osmosis/cosmwasmpool/v1beta1/genesis.ts (hunk 1 above)
Comment (jonator) [reply packages/proto-codecs/src/codegen/osmosis/cosmwasmpool/v1beta1/genesis.ts:156]: I think errors here would be unexpected

**Pull request merged**